### ISSUE TITLE
Fastlane Swift - Fix Cannot load such file -- ./fastlane/Constants/Constants

### ISSUE THREAD

**Issue by nkhanh44** [nimblehq/ios-templates#399]
## Issue

- Danger cannot load the file `./fastlane/Constants/Constants` error.

![Screen Shot 2022-11-29 at 23 10 33](https://user-images.githubusercontent.com/25881847/205216866-f2cce984-a917-407c-a3b8-590e9b29fcfc.png)
 
## Expected

In Dangerfile:
![Screenshot 2022-12-02 at 11 53 29](https://user-images.githubusercontent.com/25881847/205217654-307abd3c-1a29-47a7-9fbe-3b89fbdbafbd.png)

- Set it to the correct path `./fastlane/Constants/Constant`.
 
## Steps to reproduce

1. From the migrate-develop branch
2. Push a PR and observe the error result.

**Comment by ducbm051291** [comment]
Danger requires Constants.rb and DeliverableConstants.rb to run but in the middle of migrating Fastlane Swift, we replaced them with Constant.swift. So let's restore those two files and finish migrating Fastlane Swift. 

While investigating this case, I found that we're missing replace two lanes in the pull request review workflow.

![image](https://user-images.githubusercontent.com/24598204/208075109-7d270287-4c78-439c-8a50-09c69cd8e261.png)

![image](https://user-images.githubusercontent.com/24598204/208075124-320585b4-207a-40c0-ab6d-60d60e3d2a58.png)
